theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-placeholder-widget.tsx

```diff
@@ -94,10 +94,11 @@ export class TraceExplorerPlaceholderWidget extends ReactWidget {
                 text: ''
             }).then(async progress => {
                 progress.report({ message: 'Launching trace server... ', work: { done: 10, total: 100 } });
+                const { path, args } = this;
                 try {
-                    const resolve = await this.traceServerConfigService.startTraceServer(this.path, this.port);
+                    const resolve = await this.traceServerConfigService.startTraceServer({ path, args });
                     if (resolve === 'success') {
-                        progress.report({ message: `Trace server started on port: ${this.port}.`, work: { done: 100, total: 100 } });
+                        progress.report({ message: `Trace server started using the following arguments:  ${this.args}.`, work: { done: 100, total: 100 } });
```

Comment:
If the ${this.args} is empty then the message is: `Trace server started using the following arguments: .`

If `this.args` is empty the message should be clearer.